src/components/Messaging/Messaging.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const Messaging = () => {
+
+    return (
+        <>
+            <p>This is the Messaging page.</p>
+            <Link to="/home">Home</Link>
```

Comment:
Can we link to Profile too please from here? I imagine profile should be able to be reached by messages page

Reply:
Trying to figure out how to update this PR with my new commit. I made the commit but I'm not sure it updated the PR.